hicommonwealth/commonwealth
# Contests policy contract helpers

## Link to Issue
Closes: #7052

## Description of Changes
- Creates contestHelper lib 
- includes implementations for addContent and voteContent
- Includes a getter for general contest info(can add more if requested for product needs
- web3 instantiation with private key for relaying transactions 

## Test Plan
- Use contest the following parameters 
    - contest: `0xDf4305F7B57673c45e8A1AE726C6c2778645472d`
    - rpc: `https://base-sepolia.g.alchemy.com/v2/{API_KEY}` - get api key from alchemy or ask me for it
    - Set env `PRIVATE_KEY=` to a private key loaded with base-sepolia ETH, happy to assist with any questions related to this
- Above should allow for all functions to be tested

## Deployment Plan

1. A private key will need to be securely injected via env var on the platform(or an even more secure system if deemed necessary) 
2. Private key will need funds sent to pub key for gas on relevant networks 

## Other Considerations

- 

Example usage
<img width="832" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/36428666/51257f52-416f-4ded-b02e-14b27823e11d">
